docs/install-on-aws.md

```diff
@@ -0,0 +1,233 @@
+
+| Type            | Protocol  | Port Range | Source        | Allows            |
+| --------------  | -----     | -----      | -----         | -----             |
+| SSH             | TCP       | 22         | 0.0.0.0/0     | SSH connection    |
+| HTTP            | TCP       | 80         | 0.0.0.0/0     | Container images  |
+| Custom TCP Rule | TCP       | 2375       | 172.30.0.0/24 | Non-TLS traffic   |
+| Custom TCP Rule | TCP       | 8500       | 172.30.0.0/24 | Consul discovery  |
+| Custom TCP Rule | TCP       | 4000       | 172.30.0.0/24 | Swarm HA managers |
+
+This example doesn't use the following rules. However, they are included here for your reference.
+
+ If you create an overlay network to serve as an isolated virtual network for the Swarm cluster, add the following rules to the security group
+
+| Type             | Protocol | Port Range | Source        | Allows            |
+| --------------   | -----    | -----      | -----         | -----             |
+| Custom UDP Rule  | UDP      | 4789       | 172.30.0.0/24 | Overlay network data plane (VXLAN) |
+| Custom TCP Rule  | TCP      | 7946       | 172.30.0.0/24 | Overlay network control plane |
+| Custom UDP Rule  | UDP      | 7946       | 172.30.0.0/24 | Overlay network control plane |
+
+## Create your hosts
+
+Here, you create five Linux hosts that are part of the "Docker Swarm Example" security group and install Docker Engine on each one.
+
+Open the EC2 Dashboard and launch four EC2 instances one at a time:
+
+- During **Step 1: Choose an Amazon Machine Image (AMI)**, pick the *Amazon Linux AMI*.
+
+- During **Step 5: Tag Instance**, under **Value**, give each instance one of these names:
+    - manager0 & consul0
+    - manager1
+    - node0
+    - node1
+
+
+- During **Step 6: Configure Security Group**, choose **Select an existing security group** and pick "Docker Swarm Example".
+
+Review and launch your instances.
+
+## Install Docker Engine on each instance
+
+Connect to each instance using SSH and install Docker Engine.
+
+Update the yum packages, and keep an eye out for the "y/n/abort" prompt:
+
+        $ sudo yum update
+
+Run the installation script:
+
+        $ curl -sSL https://get.docker.com/ | sh
+
+Configure and start Docker Engine so it listens for Swarm nodes on port 2375 :
+
+        $ sudo docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock
+
+Verify that Docker Engine is installed correctly:
+
+        $ sudo docker run hello-world
+
+The output should display a "Hello World" message and other text without any error messages.
+
+Give the ec2-user root privileges:
+
+        sudo usermod -aG docker ec2-user
+
+ Then, enter `logout`.
+
+> Troubleshooting: If entering a docker command produces a message asking whether docker is available on this host, it may be because the user doesn't have root privileges. If so, use `sudo` or give the user root privileges.
```

Comment:
Docker the product is capitalized. - where as the `docker` command, is _generally_ put into code